src/ImageDetails.jsx

```diff
@@ -34,7 +34,7 @@ const ImageDetails = ({ containers, image, showAll }) => {
                 <DescriptionListDescription><ImageUsedBy containers={containers} showAll={showAll} /></DescriptionListDescription>
             </DescriptionListGroup>
             }
-            {image.Ports.length !== 0 &&
+            {image.Ports && image.Ports.length !== 0 &&
```

Comment:
Is this related? have you seen this failing that `Ports` were not defined? If so, it should be a separate commit (possibly separate PR).